quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/jaxrs/QuarkusKeycloakApplication.java

```diff
@@ -77,16 +78,17 @@ protected void loadConfig() {
     protected void createTemporaryAdmin(KeycloakSession session) {
         var adminUsername = getOption(BootstrapAdminOptions.USERNAME.getKey(), KEYCLOAK_ADMIN_ENV_VAR);
         var adminPassword = getOption(BootstrapAdminOptions.PASSWORD.getKey(), KEYCLOAK_ADMIN_PASSWORD_ENV_VAR);
+        var isTemporary = Boolean.parseBoolean(Configuration.getOptionalKcValue(BootstrapAdminOptions.EXPIRATION.getKey()).orElse("true"));
```

Comment:
This should not reference IS_TEMPORARY correct? 

The EXPIRATION option was never implemented - it still might at some point, which will cause some additional validation with the IS_TEMPORARY flag.

Reply:
You're eagle-eyed @shawkins ! Thanks for spotting this!